docs/xdcchain/developers/node_operators/masternode.md

```
@@ -519,11 +519,11 @@ After deleting the XDC folder, you can now proceed to download and apply the lat
 
 To download the XDC Network Mainnet snapshot, use the appropriate command based on the type of node you’re setting up:
 
-- **For a Full Node (latest state only):** https://download.xinfin.network/xdcchain.tar
+- **For a Full Node (latest state only):** https://rpc.xdc.network/snapshots/mainnet/full/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)